docs/src/howto-guides/023_smartrest_templates.md

```diff
@@ -0,0 +1,72 @@
+# How to add C8Y SmartRest Templates
+
+[SmartRest Templates](https://cumulocity.com/guides/10.6.0/reference/smartrest/#templates) can be used to add custom templates to the SmartRest protocol.
+
+thin-edge.io supports topic subscription to custom templates as per [C8Y documentation](https://cumulocity.com/guides/10.6.6/users-guide/device-management/#smartrest-templates).
+
+All configured templates are going to generate following MQTT subscriptions:
+
+```plain
+s/dc/template-name
+s/uc/template-name
+```
```

Comment:
```suggestion
A template named `$TEMPLATE_NAME` is requires the following subscriptions to be added when connecting Cumulocity:

```plain
s/dc/$TEMPLATE_NAME
s/uc/$TEMPLATE_NAME
``

__This is not done automatically and the custom templates have to be declared using the `tedge` command__.
````